decidim-core/app/helpers/decidim/omniauth_helper.rb

```diff
@@ -13,17 +13,17 @@ def normalize_provider_name(provider)
 
     # Public: icon for omniauth buttons
     def oauth_icon(provider)
-      info = current_organization.enabled_omniauth_providers[provider.to_sym]
+      provider_info = current_organization.enabled_omniauth_providers[provider.to_sym]
 
       name = normalize_provider_name(provider)
       name = "twitter-x" if provider == :twitter
       name = "#{name}-fill"
 
-      if info
-        icon_path = info[:icon_path]
+      if provider_info
+        icon_path = provider_info[:icon_path]
         return external_icon(icon_path) if icon_path
```

Comment:
This particular line is forcing you to set the path in the admin ( even if the default is not changed). 

To reproduce, visit the system panel, and remove the icon path for any of the enabled providers. 

Fixing this (in my case i have removed facebook icon) I get this layout, and it could be improved. (ex: have the color of SVG same as text color) 

![image](https://github.com/user-attachments/assets/2d54221c-b439-498d-be15-6ba450e3fd02)

Reply:
I'm not sure the SVG color has an element which can change its color once the SVG is removed...

Reply:
actually, having this: 
![image](https://github.com/user-attachments/assets/664f3371-d31f-4157-9b97-9ea76f788708)

result into this: 
![image](https://github.com/user-attachments/assets/96e2eda1-28f5-4361-ae9c-05510599d9ba)

Reply:
I'm not sure what this error is but all links are redirecting as they should. Can you provide more context or steps? Is this also something noticeable in develop?

Reply:
@greenwoodt I am saying that if you do not fill in the icon path in the system panel, then the `oauth_icon` will raise the error below. 
Basically, you are calling `external_icon(icon_path)`, but `icon_path` is an empty string.

Reply:
@alecslupu see the following changes in commit: https://github.com/decidim/decidim/pull/13481/commits/8ab589c3936d2851f1c8f70e2c155fbf4d2acffc with updated RELEASE_NOTES in: https://github.com/decidim/decidim/pull/13481/commits/cffc26a154144d4dc5b299127c7c58714a64d05c